app-libs/stf/src/trusted_call.rs

```diff
@@ -365,15 +368,18 @@ where
 				shield_funds(who, value)?;
 
 				// Send proof of execution on chain.
-				calls.push(ParentchainCall::Integritee(OpaqueCall::from_tuple(&(
-					node_metadata_repo
-						.get_from_metadata(|m| m.publish_hash_call_indexes())
-						.map_err(|_| StfError::InvalidMetadata)?
-						.map_err(|_| StfError::InvalidMetadata)?,
-					call_hash,
-					Vec::<itp_types::H256>::new(),
-					b"shielded some funds!".to_vec(),
-				))));
+				calls.push(ParentchainCall::Integritee {
+					call: OpaqueCall::from_tuple(&(
+						node_metadata_repo
+							.get_from_metadata(|m| m.publish_hash_call_indexes())
+							.map_err(|_| StfError::InvalidMetadata)?
+							.map_err(|_| StfError::InvalidMetadata)?,
+						call_hash,
+						Vec::<itp_types::H256>::new(),
+						b"shielded some funds!".to_vec(),
+					)),
+					mortality: GenericMortality::immortal(),
```

Comment:
Why is this one immortal? Because it is still a WIP?

Reply:
exactly. done now